### ISSUE TITLE
Looking Glass Window Not Positioned Properly on Portrait

### ISSUE THREAD

**Issue by jhon** [CaptainProton42/GodotHoloPlay#1]
Tried the demo on my LKG portrait and doesn't look like it's working out of the box. I tried quickly searching through device indexes to see if for some reason it wasn't on index 0, but no luck on any of them.

The debug output from these looks like they come directly from an SDK call with the device index so I'm surprised to see them fail, but I was unable to try to build/debug per https://github.com/CaptainProton42/GodotHoloPlayNative/issues/1

**Steps to Reproduce:**
1. Setup/attach/verify that a looking glass portrait is connected to the computer
2. Clone the GodotHoloPlayDemo repo
3. Open the project in Godot
4. Run the default scene

**Expected Results:**
Cube appears on the Looking Glass Portrait

**Actual Results:**
Window appears in the top right of the main computer display. Debug output looks like it cannot get the device properties correctly:

```--- Debugging process started ---
Godot Engine v3.3.2.stable.official - https://godotengine.org
OpenGL ES 3.0 Renderer: NVIDIA GeForce RTX 2070 with Max-Q Design/PCIe/SSE2
OpenGL ES Batching: ON
 
Updated device properties: 
win_x: 0
win_y:0
screen_w:0
screen_h:0
aspect:-1.#IND
inv_view:0
red_index:0
blue_index:2
lent_pitch:-1.#IND
lent_offset:0
lent_tilt:-1.#IND
fringe:0
subpixel:1.#INF
```

**Comment by CaptainProton42** [comment]
Thank you *so* much for testing this! You have impeccable timing as well as my own Portrait just arrived today so I've finally been able to take a look at this myself and actually fix it. (Turns out I forgot to properly initialise the "app" with the service.)

I've updated all three repositories with a version that seems to at least work on my Portrait. Give it a try if you want! (I'm especially interested in performance since I'm seeing different results when rendering to the actual Portrait.)
